### PR TITLE
Feature 1

### PR DESCRIPTION
**Pull request by Javivy**
## In this milestone I:

  - [x] Create a new repository 
  - [x] Copy all the files from the todo list project
  - [x] Create the pull request for my partner to check if I'm making use or not of the best practices.

### CODE REVIEW

**Comment by shakerAbuDrais** [comment]
Great work Javier you have done great work using the es6 module 

 

but I noticed that you could enhance your code by following small steps so I left comments in your code please review and apply. good job again :heart_eyes:

File: src/modules/delete.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const task = document.querySelectorAll('.task');
+
+const todos = JSON.parse(localStorage.getItem('todo-list')) || [];
+
+const deleteTask = () => {
+  task.forEach((t) => {
```
Comment (shakerAbuDrais) [inline src/modules/delete.js]: Great work using es6 modules but I would suggest you use a clear parameter name that is meaningful so others who might work with you in the future will understand the purpose of your code.

File: src/modules/check.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+const check = (obj) => {
+  const checkbox = document.getElementById(`checkbox-${obj.index}`);
```
Comment (shakerAbuDrais) [inline src/modules/check.js]: Great work separating your js files based on functionality, but I would suggest that you avoid defining the same constant more than once, instead, you can define it globally and use it in the function scope. that will make your code look cleaner

File: src/modules/delete.js (hunk 1 above)
Comment (Javivy) [reply src/modules/delete.js]: Thank you @shakerAbuDrais For reviewing my code :))

File: src/modules/check.js (hunk 2 above)
Comment (Javivy) [reply src/modules/check.js]: Thank you! @shakerAbuDrais I'll work on that

**Review by shakerAbuDrais — APPROVED** [review]
You have done great work, nothing else to do but to merge your project congratulations

**Pull request merged**